crates/clvm-traits/src/from_clvm.rs

```diff
@@ -220,6 +220,78 @@ impl<N, D: ClvmDecoder<Node = N>> FromClvm<D> for chia_bls::Signature {
     }
 }
 
+#[cfg(feature = "chia-secp")]
+impl<D> FromClvm<D> for chia_secp::Secp256k1PublicKey
```

Comment:
I must be misremembering some details of the dependencies here. It looks backwards to have the traits crate implement the trait for these types. The dependency seems inverted.

Reply:
It's the same rationale as chia-bls:

clvmr depends on chia-bls
chia-bls would have to depend on clvm-traits
clvm-traits depends on clvmr

Reply:
right..